resources/assets/js/components/dues/DuesTransaction.vue

```diff
@@ -0,0 +1,79 @@
+<template>
```

Comment:
Should this view, after accepting a payment, show any data about the payment on the page? I think that would be useful.

Reply:
I'm thinking of the use case here where you have 1 person accepting dues from 5 or 10 people in a meeting. A confirmation page will only slow them down.

Reply:
Nah not a confirmation page. I mean on the DuesTransaction page itself when you look at it at any time, assuming the person has the appropriate permissions.